HISTORY.md

```diff
@@ -1,3 +1,52 @@
+# TileDB v2.24.0 Release Notes
+
```

Comment:
Should we add a notice about the removal of the superbuild?

We can add something like:

> In version 2.25.0, the superbuild architecture of the build system will be removed and TileDB will become a regular CMake project. Build commands of the form `make` && `make -C tiledb <targets>` will have to be replaced by `make <targets>`. You can preview the effects of this change by configuring with `-DTILEDB_CMAKE_IDE=ON`.

Reply:
I have to at least make this 2.26 to give users a chance to react. @ihnorton thoughts?

Reply:
+1. Some suggested tweaks:
```
In version 2.25.0, the superbuild architecture of the build system will be removed and TileDB will become a single-level CMake project with no CMake sub-build for libtiledb after dependencies are built (as of TileDB 2.24, dependencies are built at configuration time with vcpkg). `make install-tiledb` will continue to work, but will become effectively an alias for `make install`, which will always produce an up-to-date install after this change. Other build commands of the form `make && make -C tiledb <targets>` will have to be replaced by `make <targets>`. You can preview the effects of this change by configuring with CMake and passing the -DTILEDB_CMAKE_IDE=ON option.
```

Reply:
Agree on waiting until 2.26. It's not an API change, but good to give lead time.